src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -97,17 +101,21 @@ public static void OpenPropertiesWindow(object item, IShellPage associatedInstan
 				RequestedTheme = ThemeHelper.RootTheme
 			};
 
-			var propertiesWindow = new WinUIEx.WindowEx
+			WinUIEx.WindowEx propertiesWindow;
+			if (!WindowCache.TryTake(out propertiesWindow!))
 			{
-				IsMinimizable = false,
-				IsMaximizable = false,
-				MinWidth = 460,
-				MinHeight = 550,
-				Width = 800,
-				Height = 550,
-				Content = frame,
-				Backdrop = new WinUIEx.MicaSystemBackdrop(),
-			};
+				propertiesWindow = new();
+				propertiesWindow.Closed += PropertiesWindow_Closed;
+			}
+
+			propertiesWindow.IsMinimizable = false;
+			propertiesWindow.IsMaximizable = false;
+			propertiesWindow.MinWidth = 460;
+			propertiesWindow.MinHeight = 550;
+			propertiesWindow.Width = 800;
+			propertiesWindow.Height = 550;
+			propertiesWindow.Content = frame;
+			propertiesWindow.SystemBackdrop = new MicaBackdrop();
```

Comment:
It was initially initialized as `MicaSystemBackdrop`. Is the change wanted?

Reply:
Yes, this is necessary. I found WinUIEx's `MicaSystemBackdrop` isn't applied to cached windows. So I use WASDK's `MicaBackdrop`.

Reply:
@hez2010 this might be useful for the work you're doing.